.gitignore

```diff
@@ -4,5 +4,6 @@ _site/
 .optemp/
 _themes*/
 _repo.*/
+*.DS_Store
```

Comment:
Was this intentional?

Reply:
Yes. This is a MacOS system file that is created whenever one visits a folder in the file explorer UI. I created a workaround for this same file in another repo a few months ago. This is a typical strategy for ensuring that the file isn't tracked in git repos that MacOS users interact with.